testsuite/features/step_definitions/common_steps.rb

```diff
@@ -744,8 +744,8 @@
   when /^(opensuse|sles)/
     repos, _code = node.run('zypper lr | grep "tools" | cut -d"|" -f2')
     node.run("zypper mr --enable #{repos.gsub(/\s/, ' ')}")
-  when /^centos/
-    repos, _code = node.run('yum repolist disabled 2>/dev/null | grep "tools" | cut -d" " -f1')
+  when /^(centos|rocky)/
+    repos, _code = node.run('yum repolist disabled 2>/dev/null | grep "tools_" | cut -d" " -f1')
```

Comment:
This was done because from what I see we only want to enable client tools. On Rocky, however, we have also the `Rocky-PowerTools` repository which resulted in an error while testing without this change:

```bash
[root@rocky-min-rocky8 yum.repos.d]# pwd
/etc/yum.repos.d
[root@rocky-min-rocky8 yum.repos.d]# ll
total 76
-rw-r--r--. 1 root root  711 Oct 13 11:41 Rocky-AppStream.repo
-rw-r--r--. 1 root root  696 Oct 17 11:32 Rocky-BaseOS.repo
-rw-r--r--. 1 root root 1753 Aug 30 11:05 Rocky-Debuginfo.repo
-rw-r--r--. 1 root root  360 Aug 30 11:05 Rocky-Devel.repo
-rw-r--r--. 1 root root  696 Oct 13 09:39 Rocky-Extras.repo
-rw-r--r--. 1 root root  731 Aug 30 11:05 Rocky-HighAvailability.repo
-rw-r--r--. 1 root root  680 Aug 30 11:05 Rocky-Media.repo
-rw-r--r--. 1 root root  680 Aug 30 11:05 Rocky-NFV.repo
-rw-r--r--. 1 root root  690 Aug 30 11:05 Rocky-Plus.repo
-rw-r--r--. 1 root root  715 Aug 30 11:05 Rocky-PowerTools.repo
-rw-r--r--. 1 root root  681 Aug 30 11:05 Rocky-RT.repo
-rw-r--r--. 1 root root  746 Aug 30 11:05 Rocky-ResilientStorage.repo
-rw-r--r--. 1 root root 2335 Aug 30 11:05 Rocky-Sources.repo
-rw-r--r--. 1 root root  254 Oct 13 09:39 epel.repo
-rw-r--r--. 1 root root  262 Oct 13 14:16 susemanager:bootstrap.repo
-rw-r--r--. 1 root root  530 Oct 17 11:32 susemanager:channels.repo
-rw-r--r--. 1 root root  301 Oct 13 09:39 test_repo_rpm_pool.repo
-rw-r--r--. 1 root root  227 Oct 17 11:32 tools_pool_repo.repo
-rw-r--r--. 1 root root  194 Oct 17 11:32 tools_update_repo.repo
```